apim/4.x/build/files/graviteeio-apim-management-ui.conf

```diff
@@ -1,8 +1,36 @@
 server {
 	listen  0.0.0.0:8084;
-	root    /opt/graviteeio/apim/management-ui;
 
-	location / {
-		index index.html;
-	}
+    add_header X-Frame-Options "SAMEORIGIN" always;
+    add_header Content-Security-Policy "frame-ancestors 'self';" always;
+    add_header X-XSS-Protection "1; mode=block";
+    add_header X-Content-Type-Options nosniff;
+    add_header X-Permitted-Cross-Domain-Policies none;
+
+    index index.html index.htm;
+    charset utf-8;
+
+    gzip on;
```

Comment:
All these gzip parameters - is it APIM specific or general tuning?

Reply:
This come from setting already applied in APIM UI's docker images.